### PR TITLE
Group logic together in extensions

### PR DESCRIPTION
**Pull request by taquitos**
Make it easier to navigate iOS project by grouping together functionality in extensions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-purchases",
-  "version": "3.0.0-beta",
+  "version": "3.0.1-beta",
```
Comment (aboedo) [inline package.json:3]: I'm curious about the version number, why 3.0.1 if we haven't shipped 3.0.0 yet?

File: plugin.xml [hunk 2]
```diff
@@ -28,6 +28,11 @@
             </feature>
         </config-file>
         <source-file src="src/ios/PurchasesPlugin.swift" />
+        <source-file src="src/ios/PurchasesPlugin+APIHelpers.swift" />
+        <source-file src="src/ios/PurchasesPlugin+Attribution.swift" />
+        <source-file src="src/ios/PurchasesPlugin+PurchasesDelegate.swift" />
+        <source-file src="src/ios/PurchasesPlugin+Purchasing.swift" />
+        <source-file src="src/ios/PurchasesPlugin+Users.swift" />
```
Comment (aboedo) [inline plugin.xml:35]: yesssss

File: package.json (hunk 1 above)
Comment (taquitos) [reply package.json:3]: It makes it easier to know if the plugin was updated in the sample app - as in- all the newer code got pulled in and not just from a cache. I plan on moving this back to 3.0.0, it was just the easiest way for me to be sure my changes were coming through. Do you anticipate a problem with rev'ing this and then going back?

File: plugin.xml (hunk 2 above)
Comment (taquitos) [reply plugin.xml:35]: 🕺😄

**Pull request merged**